src/protocols/secure_channel/PASESession.cpp

```diff
@@ -803,6 +803,10 @@ CHIP_ERROR PASESession::OnMessageReceived(ExchangeContext * exchange, const Payl
     CHIP_ERROR err = ValidateReceivedMessage(exchange, payloadHeader, msg);
     SuccessOrExit(err);
 
+#if CHIP_CONFIG_SLOW_CRYPTO
+    ReturnErrorOnFailure(mExchangeCtxt->SendStandaloneAckMessage());
+#endif // CHIP_CONFIG_SLOW_CRYPTO
```

Comment:
Please send only if it's not statusreport

Reply:
fixed